src/main/java/duke/DeleteTaskMessage.java

```diff
@@ -0,0 +1,20 @@
+package duke;
+
+/**
+ * The class for creating Duke' response after a task is deleted from the list.
+ */
+public class DeleteTaskMessage extends DukeMessage{
+    private Task deletedTask;
+    private int taskIndex;
+
+    public DeleteTaskMessage(int index) {
+        this.taskIndex = index;
+        this.deletedTask = TaskList.getTaskList().getTasks().remove(taskIndex);
+    }
+    public void display() {
+        System.out.println("Theek h bhai... ye task hata diya");
+        System.out.println(deletedTask.getTaskString());
+        System.out.println("Ab " + TaskList.getTaskList().getSize() + " tasks bache list mein.");

```

Comment:
Should these be in English?

Reply:
Haha well yes... but prof said we can choose a personality and I chose an angry indian uncle... now there's no turning back